osi_object.proto

```diff
@@ -436,6 +437,84 @@ message MovingObject
         // through.
         //
         optional double ground_clearance = 6;
+
+        // Description of each wheel.
+        //
+        // \note OSI uses singular instead of plural for repeated field names.
+        //
+        repeated WheelData wheel_data = 7;
+
+        // \brief Detailed wheel data.
+        // The focus is on the description of a wheel regarding the perceivable 
+        // information from the outside. 
+        // It is not intended to be used for e.g. dynamic calculations.
+        //
+        message WheelData
+        {
+            // The axle which contains this wheel. A value of 0 represents the
+            // front-most axle of the vehicle with higher numbers incrementing
+            // towards the rear-most axle.
+            //
+            optional uint32 axle = 1;
+
+            // The index of the wheel on the axle, counting in the direction
+            // of positive-y, i.e. right-to-left.
+            //
+            // For example, on a standard 2-axle, 4-wheel car, the rear-right
+            // wheel would be (axle=1, index=0).
+            // This concept works also for twin tires.
+            //
+            optional uint32 index = 2;
+            
+            // Position of the wheel (geometric center of the wheel).
+            // The reference system is the vehicle frame (center of bounding box).
```

Comment:
Is that mean the vehicle frame is also the center of the bounding box?

Reply:
This does not mean that the vehicle frame is also **the** center of the bounding box.
According to the following [Detailed Description](https://opensimulationinterface.github.io/open-simulation-interface/structosi3_1_1MovingObject_1_1VehicleAttributes.html):
_For all vehicles, including host vehicles, the position given in MovingObject::base . BaseMoving::position points to the center of the vehicle's bounding box._

Looking at the picture below, the bounding box consists of the doted 3D lines and the reference coordinate system is the one circled in red, which lies in the center of the bounding box.

_// The reference system is the vehicle frame (center of bounding box)._ is referring to the same coordinate system.

![image](https://user-images.githubusercontent.com/25367407/99950023-3860e180-2d7c-11eb-9e8c-06dc73c0309e.png)

Reply:
I reworked that, please have a short check @kmeids  and @paagkame .
There is also a line for the picture @paagkame . Thank you very much in advance!! :)

Reply:
Good from my side!

Reply:
Good from my side, too.